### ISSUE TITLE
Crash on create new task

### ISSUE THREAD

**Issue by michaelof** [dmfs/opentasks#413]
1.1.9/Android 4.4.2

Tasks shown for several OwnCloud calendars, synced via DavDroid. All fine.

OpenTasks groups Task in list view (most left button) as default grouped by calendar/calDAV source.

Each group header shows a "+" for creation of tasks for the respective group.

--> Clicking on this "+" leads reproducible to a crash '"OpenTasks" wurde beendet'

Workaround: "General" large "+" sign, always visible, could be used without a crash, works fine. Doing this, appropriate calendar must be selected as needed.

**Comment by agilob** [comment]
```
09-18 21:41:54.187 29113-29161/? I/CurrentAppMonitor: Current app is: org.dmfs.tasks, last was: org.dmfs.tasks
09-18 21:41:54.858 7900-7900/? E/AndroidRuntime: FATAL EXCEPTION: main
                                                 Process: org.dmfs.tasks, PID: 7900
                                                 java.lang.ClassCastException: android.support.v7.widget.TintContextWrapper cannot be cast to android.support.v4.app.FragmentActivity
                                                     at org.dmfs.tasks.groupings.ByList$2$1.onClick(ByList.java:281)
                                                     at android.view.View.performClick(View.java:4640)
                                                     at android.view.View$PerformClick.run(View.java:19425)
                                                     at android.os.Handler.handleCallback(Handler.java:733)
                                                     at android.os.Handler.dispatchMessage(Handler.java:95)
                                                     at android.os.Looper.loop(Looper.java:146)
                                                     at android.app.ActivityThread.main(ActivityThread.java:5593)
                                                     at java.lang.reflect.Method.invokeNative(Native Method)
                                                     at java.lang.reflect.Method.invoke(Method.java:515)
                                                     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1283)
                                                     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1099)
                                                     at dalvik.system.NativeStart.main(Native Method)
09-18 21:41:54.858 850-8384/? W/ActivityManager:   Force finishing activity org.dmfs.tasks/.TaskListActivity```

**Comment by brotkiste** [comment]
+1

**Comment by ektus** [comment]
Same here. Both programs installed from f-droid, directly creating a local task works, directly creating a task on the server fails with crash, creating a task with the orange task button and switching to the server in the new task settings works.

**Comment by JaapPutman** [comment]
+1

**Comment by dmfs** [comment]
Fixed in 1.1.10.
